### ISSUE TITLE
Lists of links should be marked up as unordered lists

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#5321]
Lists of links are much easier to navigate in a screenreader if they are marked up as lists. We should mark up the following as lists:
- Zooniverse site navigation.
- user profile dropdown menu.
- project navigation menu.
- lists of links in the footer.

[`aria-current`](https://tink.uk/using-the-aria-current-attribute/) can be used to highlight the currently item in a menu too eg. which page you are currently on.
